content/php/concepts/string-functions/terms/html-entity-decode/html-entity-decode.md

```diff
@@ -0,0 +1,43 @@
+---
+Title: 'PHP html_entity_decode() Function'
+Description: 'Converts HTML entities to characters.'
+Subjects: 
+- 'Web Design'
+- 'Web Development'
+Tags:
+- 'Functions'
+- 'Characters'
+CatalogContent:
+- 'php/concepts/string-functions/terms/html-entity-decode'
```

Comment:
```suggestion
- 'php/concepts/string-functions/terms/html-entity-decode'
---
```
Close the meta data after this please

Reply:
There are: '---' below your comment. Let me know if this is enough.